### ISSUE TITLE
Dump of taxon IDs with at least one image

### ISSUE THREAD

**Issue by jhammock** [EOL/tramea#330]
For users hitting our pages API for images, a fairly common use case. From @hyanwong:

...a dump of taxon ids (EoL page ids) which have at least one image (of any sort) associated with them. Then I could at least limit my API calls to those taxa I knew had a chance of returning something. I'm envisaging this dump as simply a long list of pageIDs.

**Comment by hyanwong** [comment]
Thanks Jen. As a start, I wonder what the percentage of taxon pages with no images is. Is this on the site stats somewhere?

**Comment by jhammock** [comment]
Yes! http://eol.org/statistics  but you may not want a proportion, because our total taxon pages includes a lot of bad names...

**Comment by hyanwong** [comment]
Thanks. So about 10% have an image, but many of the 90% may be false negatives.

FWIW, I'm finding that of 589682 species-level taxa I've queried so far, only 26367 (~5%) have a pd, cc-by, or cc-by-sa image. I suspect 2 things are going on here: many taxa only have cc-xxx-nc images, and also the EoL total includes higher-level taxa (e.g. genera) that are more likely to have images, as they collect all the images of their descendants.

Either way, that says to me that I'll probably see a 10-fold reduction in number of queries I need to make if I can get hold of a list of taxon pages with at least one image, as per this issue.

**Comment by JRice** [comment]
Within what time-frame would that information be useful?

**Comment by hyanwong** [comment]
Well, there's no major hurry. I'm happy chugging along querying every one of my 2.1 million OpenTree species, but at the current rate it'll take 10 days. If I can restrict the query, I'm hoping it will only take a day. 

I foresee this as an ongoing process of updating my tree with images, so once I get to the end, I'll probably cycle round again to look for updates. But I'm aware that I'm causing load / traffic for EoL, so am looking at ways I could run this less regularly, e.g. once every 10 days, rather than continuously.
